README.md

```diff
@@ -61,18 +61,18 @@ sudo make install
 with some of them enabled by default (see `BUILDTAGS` in top-level `Makefile`).
 
 To change build tags from the default, set the `BUILDTAGS` variable for make,
-e.g.
+e.g. to disable selinux:
```

Comment:
to disable seccomp, right?

Reply:
guess it's too late to change to a `noseccomp` build-tag instead of `seccomp` ? Would me more intuitive (and make the logic easier in the makefile etc)

Reply:
Ah yeah, seccomp. Fixed!

Reply:
So, `seccomp` vs `noseccomp` is out of scope for this PR, and is probably post-1.0 material. Should be trivial to implement though, as no third-party packages use it.